EmpleoDotNet/Views/JobOpportunity/Wizard.cshtml

```diff
@@ -95,8 +95,28 @@
                     @Html.LabelFor(m => m.Description)
                     @Html.TextAreaFor(m => m.Description, new { @class = "form-control textarea", @data_parsley_group="job-details", @data_parsley_errors_container="#description_errors", required="" })
                     @Html.ValidationMessageFor(m => m.Description)
-                    <span><b>Tip:</b> Se lo mas explicito posible, si eres de recursos humanos consulta con una persona de 
-                    tecnolog&iacute;a sobre cuales son los requerimientos t&eacute;cnicos.</span>
+                    <span><b>Tip:</b> Describa las actividades y actitudes que se esperan de quien ocupa este puesto de trabajo.</span>
+                    <span id="description_errors" class="danger"></span>
+                </div>
+
+                <div class="form-group">
+                    @Html.LabelFor(m => m.Requirements)
+                    @Html.TextAreaFor(m => m.Requirements, new { @class = "form-control textarea", @data_parsley_group = "job-details", @data_parsley_errors_container = "#description_errors", required = "" })
+                    @Html.ValidationMessageFor(m => m.Requirements)
+                    <span>
+                        <b>Tip:</b> Se lo mas explic&iacute;to posible, si eres de recursos humanos consulta con una persona de
+                        tecnolog&iacute;a sobre cuales son los requerimientos t&eacute;cnicos.
```

Comment:
It's not big deal, but if you are using HTML5 and utf-8 encoding you can put the raw accents in the html. Is just for info. I see everything fine.